platforms/native/platform-native/src/test/groovy/org/gradle/nativeplatform/toolchain/internal/NativeCompilerTest.groovy

```diff
@@ -166,7 +166,7 @@ abstract class NativeCompilerTest extends Specification {
         sourceFiles.each { sourceFile ->
             1 * commandLineTool.execute(_, _)
         }
-        4 * timeProvider.getCurrentTime()
+        2 * timeProvider.getCurrentTime()
```

Comment:
💭 I'm not 100% sure why we are calling this fewer times, but it doesn't seem to make an actual difference.

Reply:
Yep, fixes it, thanks.